Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -197,17 +197,17 @@ extension Module  {
 
     /// Converts build settings dictionary to a Plist object.
     ///
-    /// Adds string values in dictionaries as is and array values are quoted and then converted
-    /// to a string joined by whitespace.
+    /// Adds string values in dictionaries as is and array values are converted to a
+    /// string joined by whitespace.
     private func toPlist(_ buildSettings: [String: Any]) -> Plist {
         var buildSettingsPlist = [String: Plist]()
         for (k, v) in buildSettings {
             switch v {
             case let value as String:
                 buildSettingsPlist[k] = .string(value)
             case let value as [String]:
-                let escaped = value.map { "\"" + Plist.escape(string: $0) + "\"" }.joined(separator: " ")
-                buildSettingsPlist[k] = .string(escaped)
+                let joined = value.joined(separator: " ")
```

Comment:
Nope how build settings works is: arrays are separated by a space which is serialised as a plist string. Values in that array which needs escaping are escaped using quotes. I think this can be fixed by quoting only those objects of `value` array which contains a space, quote or escape character and then everything just joined by space. However, @abertelrud can better suggest what is a good approach here.